app.js

```diff
@@ -281,6 +237,58 @@ d.run(function () {
 			});
 		}],
 
+		webSocket: ['config', 'connect', 'logger', 'network', function (scope, cb) {
+			var webSocketConfig = {
+				workers: 1,
+				port: scope.config.port,
+				wsEngine: 'uws',
+				appName: 'lisk',
+				workerController: workersController.path,
+				perMessageDeflate: false,
+				secretKey: 'liskSecretKey',
```

Comment:
But for what we need that secretKey, even default? Is used to encrypt connection, or what?